src/main/resources/db/migration/V1_2__visit_id.sql

```diff
@@ -0,0 +1,9 @@
+create table visit_id
+(
+    nomis_id bigint       not null  constraint visit_id_nomis_id_pkey   primary key,
+    vsip_id  varchar(80)  not null  constraint visit_id_vsip_id_unique  unique,
+    label    varchar(20),
+    source   varchar(20)  not null
+);
+
+create index visit_id_label_index on visit_id (label);
```

Comment:
not sure sorting a `varchar` will work as expected?  There doesn't seem to be any constraint on the format of the timestamp coming in - will it be `DD/MM/YYYY` etc?  Might be better to just make it a timestamp field in the database if that's what we're storing.

Reply:
I wasnt sure what we will be doing yet. I would expect an ISO timestamp trhough?